netlib/http/cookies.py

```diff
@@ -161,10 +175,11 @@ def _parse_set_cookie_pairs(s):
 def parse_set_cookie_headers(headers):
     ret = []
     for header in headers:
-        v = parse_set_cookie_header(header)
-        if v:
-            name, value, attrs = v
-            ret.append((name, SetCookie(value, attrs)))
+        cookies = parse_set_cookie_header(header)
+        if cookies:
+            for cookie in cookies:
+                name, value, attrs = cookie
```

Comment:
I think you can directly use this as loop args: `for name, value, attrs in cookies:`